### ISSUE TITLE
Some recipes are busted (fabric), easy fixes

### ISSUE THREAD

**Issue by dehen** [PTOM76/small-stairs119#2]
1. Both nether brick recipes need to have the ingredient changed from "brick" to "bricks". Pretty sure there's no singular of "red_nether_bricks":
```
  "key": {
    "#": {
      "item": "minecraft:nether_bricks"
    }
```

2. For both nether brick varients the output needs to be singular:
```
  "result": {
    "item": "smallstairs:small_stairs_red_nether_brick",
    "count": 6
  }
```

3. Dark oak is referencing Crimson for the whole recipe/output. Crimson references spruce. Spruce references itself, so that's good, and probably the end of this misalignment? Obviously the fix is to aim them at the right materials.

**Comment by PTOM76** [comment]
I fixed it and uploaded 1.0.2.

**Comment by dehen** [comment]
Thanks!
